app/components/Editor.tsx

```diff
@@ -280,7 +280,16 @@ function Editor(props: Props, ref: React.RefObject<SharedEditor> | null) {
           ref={mergeRefs([ref, handleRefChanged])}
           uploadFile={onUploadFile}
           onShowToast={showToast}
-          embeds={embeds}
+          embeds={embeds.map((e) => {
+            const em = find(
+              props.embedIntegrations,
+              (i) => i.service === e.component.name.toLowerCase()
+            );
+            return new EmbedDescriptor({
+              ...e,
+              host: em?.settings.hostname,
```

Comment:
I'd suggest we spread all settings here, `hostname` won't be the only one needed in the long run. Also should choose – `host` or `hostname`, host is better but technically doesn't include the protocol so probably just use `url` is the closest to accurate?

Reply:
Agree. `url` makes more sense. Will update that.

Reply:
Fixed!